micro-rdk/src/common/robot.rs

```diff
@@ -740,6 +740,23 @@ mod tests {
                             Kind::StringValue("13".to_owned()),
                         ]),
                     ),
+                    (
+                        "digital_interrupts".to_owned(),
+                        Kind::VecValue(vec![
+                            Kind::StructValue(HashMap::from([(
+                                "pin".to_owned(),
+                                Kind::StringValue("13".to_owned()),
+                            )])),
+                            Kind::StructValue(HashMap::from([(
+                                "pin".to_owned(),
+                                Kind::StringValue("14".to_owned()),
```

Comment:
should it be NumberValue? Probably doesn't matter anyway

Reply:
I just checked, either works, note that `pins` should also use `NumberValue(f64)`.

Will change these for digital interrupts to use `NumberValue`, then we'd indirectly catch any issues with the `Kind` conversion